### DISCUSSION TITLE
"disable_splash_rendering" question

### DISCUSSION THREAD

**Discussion by Aqa-Ib** [hyprwm/Hyprland#4091] [Q&A] [answered]
I do not notice what "disable_splash_rendering = true" is doing. What is the 'splash' in hyprland ? Thanks for your attention.

**Comment by vaxerski** [comment, marked as answer]
![image](https://github.com/hyprwm/Hyprland/assets/43317083/bb839c35-4388-4d89-a5e1-eebc5f27aa10)
